packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -68,7 +68,7 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
 
     this.state = {
       activeDescendantId: getId('DatePickerDay-active'),
-      weeks: this._getWeeks(navigatedDate, selectedDate, today)
+      weeks: this._getWeeks(props.navigatedDate, props.selectedDate, props.dateRangeType, props.firstDayOfWeek, today)
```

Comment:
Why add the `props.` reference here, rather than using the destructive assignment above on line 67?